debug-init-env.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2015 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+DATADIR=data
+
+touch ${DATADIR}/easy-archive-bag.log
+
+echo "Creating staging area in $DATADIR..."
+mkdir -p ${DATADIR}/easy-archive-bag-staging
+
+echo "Copy bags to $DATADIR..."
+mkdir ${DATADIR}/bags
+cp -r src/test/resources/bags/basic-sequence-unpruned ${DATADIR}/bags/basic-sequence-unpruned
+
+echo "OK"
```

Comment:
An OK might be misleading if the mkdir's or copy's fail, or am I missing some magic trick that aborts the script on an error? These checks are also missing in the [archetype](https://github.com/DANS-KNAW/easy-module-archetype/blob/master/src/main/resources/archetype-resources/debug-init-env.sh).